src/pages/CategorizationPage.js

```diff
@@ -10,30 +9,13 @@ const Container = styled.div`
   margin: 0 auto;
 `;
 
-const CategorizationPage = ({ userState, updateCategories }) => (
+const CategorizationPage = () => (
   <Container>
     <Grid container>
       <Typography variant="h2">Categorization</Typography>
-      {!userState ? (
-        <Typography variant="h4">Select a file and bank on import page</Typography>
-      ) : (
-        <Categorizer userState={userState} updateCategories={updateCategories} />
-      )}
+      <Categorizer />
```

Comment:
Should we here fetch the userState from the store and check if its not defined then we display the header?

Reply:
Yes, then we can add the same thing also in the VisualizationPage